README.md

```diff
@@ -35,19 +35,19 @@ nvm use 6.16.0
 
 Build TAU by running the following command in the tau directory:
 
-`npx grunt build`
+`grunt build`
 
 ## Running the tests
 
 TAU has automated tests. To run tests execute command:
 
-`npx grunt test`
+`grunt test`
 
 ## Debuggging
 
 In order to set additional logging in TAU you need to build TAU with --tau-debug option
 
-`npx grunt build --tau-debug`
+`grunt build --tau-debug`
```

Comment:
npx grunt build --tau-debug
I think should stay.
AFAIK I can not pass argument to npm command so 
npm run build --tau-debug won't work

Reply:
Done.